CMakeLists.txt

```diff
@@ -113,22 +125,26 @@ else( Dripline_BUILD_PYTHON )
     #list( REMOVE_ITEM PRIVATE_EXT_LIBS Python::Python )
 endif( Dripline_BUILD_PYTHON )
 
+# all parts of dripline use Scarab, so include it here
+pbuilder_use_sm_library( Scarab Scarab )
+
 
 ##################
 # Build Dripline #
 ##################
 
-if( Dripline_OFFLINE )
-    add_definitions( -DDL_OFFLINE )
-else()
-    remove_definitions( -DDL_OFFLINE )
-endif()
 
-add_definitions( -DDL_MAX_PAYLOAD_SIZE=${Dripline_MAX_PAYLOAD_SIZE} )
+#add_library( ${PROJECT_NAME}_void INTERFACE )
+#export( TARGETS ${PROJECT_NAME}_void
+#    FILE ${PROJECT_BINARY_DIR}/${PROJECT_NAME}Targets.cmake
+#)
+#install( TARGETS ${PROJECT_NAME}_void 
+#    EXPORT ${PROJECT_NAME}Targets 
+#    LIBRARY DESTINATION ${LIB_INSTALL_DIR} 
+#)
```

Comment:
is this coming back? maybe go ahead and remove?

Reply:
Done